app/controllers/authenticate_controller.rb

```diff
@@ -4,6 +4,23 @@ class AuthenticateController < ApplicationController
   include BasicAuthenticator
   include AuthorizeResource
 
+  def oidc_authenticate_token
+    params.permit!
+
+    auth_token = Authentication::Handler::AuthenticationHandler.new(
+      authenticator_type: params[:authenticator]
+    ).call(
+      parameters: params.to_hash.symbolize_keys,
+      request_ip: request.ip
+    ) do |authenticator|
+      Authentication::AuthnOidc::V2::Strategies::Token.new(
+        authenticator: authenticator
+      ).callback(request.authorization.to_s.split(' ').last || request.raw_post)
```

Comment:
I think we should verify that the first value of the `split` is "Bearer" and not just any word. To enforce the correct standard. WDYT?